### PR TITLE
upgrade broccoli-babel-transpiler to stable 6.0.0

### PR DESCRIPTION
**Pull request by baweaver**
Noticed that `broccoli-babel-transpiler` has gone 6.0.0 full release fairly recently.

Side note, any chance of upgrading it for the stable 2.12 branches, or is that a moot point with 2.13 coming up?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "babel-plugin-transform-es2015-modules-amd": "^6.24.0",
     "bower-config": "^1.3.0",
     "bower-endpoint-parser": "0.2.2",
-    "broccoli-babel-transpiler": "6.0.0-alpha.3",
+    "broccoli-babel-transpiler": "6.0.0",
```
Comment (rwjblue) [inline package.json]: Can you swap this to ^6.0.0? Also, could you update the target branch to be `beta`?

**Comment by baweaver** [comment]
...well, noted not to trust editing things on git again.

**Comment by Turbo87** [comment]
@baweaver it appears that this PR includes a few more commits than you intended. could you rebase this PR?

**Comment by Turbo87** [comment]
this seems resolved now that https://github.com/ember-cli/ember-cli/pull/7023 is merged

**Pull request closed without merging**